packages/payment-integration-api/src/hosted-form/hosted-form-options.ts

```diff
@@ -0,0 +1,52 @@
+//copied from packages/core/src/hosted-form/hosted-form-options.ts
```

Comment:
Do we need to have this comment? Also shall we remove the interfaces in core package in this case?  Since hosted form is a payments thing so we should be able to remove that.

cc @davidchin

Reply:
We can remove it in another PR as a quick follow-up to keep this PR simple.